tests/suites/test_suite_bignum_random.function

```diff
@@ -158,6 +191,80 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void mpi_mod_random_values( int min, char *max_hex, int rep )
+{
+    /* Same RNG as in mpi_core_random_basic */
```

Comment:
Other places that say this have

```
mbedtls_test_rnd_pseudo_info rnd_core = rnd_pseudo_seed;
```

where

```
const mbedtls_test_rnd_pseudo_info rnd_pseudo_seed = {
    /* 16-word key */
    {'T', 'h', 'i', 's', ' ', 'i', 's', ' ',
     'a', ' ', 's', 'e', 'e', 'd', '!', 0},
    /* 2-word initial state, should be zero */
    0, 0};
```

which is different to this. Does it matter? Can we just fix it in a second, quick, PR?

Reply:
It matters to have confidence that we're testing values() operation in cases where `random()` ends up giving up because it's retried too many times. This confidence comes from running values() and basic() with the same RNG and same bounds. This is the earlier, pre-acdefdd51ac8defc0637b7b996d8976229a1463b seed from basic(), which I forgot to update when I rebased this branch after the updates to #6303. I'll fix it.